Functions.Templates/Templates/CosmosDBTrigger-CSharp-Isolated/CosmosDBTriggerCSharp.cs

```diff
@@ -3,41 +3,40 @@
 using Microsoft.Azure.Functions.Worker;
 using Microsoft.Extensions.Logging;
 
-namespace Company.Function
+namespace Company.Function;
+
+public class CosmosDBTriggerCSharp
 {
-    public class CosmosDBTriggerCSharp
-    {
-        private readonly ILogger _logger;
+    private readonly ILogger _logger;
 
-        public CosmosDBTriggerCSharp(ILoggerFactory loggerFactory)
-        {
-            _logger = loggerFactory.CreateLogger<CosmosDBTriggerCSharp>();
-        }
+    public CosmosDBTriggerCSharp(ILoggerFactory loggerFactory)
```

Comment:
Recommend primary constructor and also switching to `ILogger<CosmosDBTriggerCSharp>`